rct229/rulesets/ashrae9012019/section19/section19rule15.py

```diff
@@ -0,0 +1,213 @@
+            each_rule=Section19Rule15.HVACRule(),
+            index_rmr="baseline",
+            id="19-15",
+            description="For baseline system types 9 & 10, the system design supply airflow rates shall be based on the temperature difference between a supply air temperature set point of 105°F and the design space-heating temperature set point, the minimum outdoor airflow rate, or the airflow rate required to comply with applicable codes or accreditation standards, whichever is greater.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1.2.8.2",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_instances/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    def create_data(self, context, data):
+        rmi_b = context.baseline
+        rmi_p = context.proposed
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+
+        hvac_info_dict_b = {}
+        for hvac_b in find_all(
+            "$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+            rmi_b,
+        ):
+            hvac_id_b = hvac_b["id"]
+            hvac_info_dict_b[hvac_id_b] = {}
+            hvac_info_dict_b[hvac_id_b]["all_design_setpoints_105"] = True
+            hvac_info_dict_b[hvac_id_b]["proposed_supply_cfm"] = ZERO.FLOW
+            hvac_info_dict_b[hvac_id_b][
+                "are_all_hvac_sys_fan_objs_autosized"
+            ] = are_all_hvac_sys_fan_objs_autosized(rmi_b, hvac_id_b)
+
+            zones_and_terminal_units_served_by_hvac_sys_dict = (
+                get_dict_of_zones_and_terminal_units_served_by_hvac_sys(hvac_b)
+            )
+
+            for terminal_b in zones_and_terminal_units_served_by_hvac_sys_dict[
+                "terminal_unit_list"
+            ]:
+                for terminal in find_all(
+                    f'$.buildings[*].building_segments[*].zones[*].terminals[?(@.id == "{terminal_b}")]',
+                    rmi_b,
+                ):
+                    if (
+                        getattr_(
+                            terminal,
+                            "Terminal",
+                            "supply_design_heating_setpoint_temperature",
+                        )
+                        != REQ_DESIGN_SUPPLY_AIR_TEMP_SETPOINT
+                    ):
+                        hvac_info_dict_b[hvac_id_b]["all_design_setpoints_105"] = False
+
+            for zone_b in zones_and_terminal_units_served_by_hvac_sys_dict["zone_list"]:
+                for terminal_p in find_one(
+                    f'$.buildings[*].building_segments[*].zones[?(@.id == "{zone_b}")].terminals[*]',
+                    rmi_p,
+                ):
+                    hvac_info_dict_b[hvac_id_b]["proposed_supply_cfm"] += getattr_(
+                        terminal_p, "Terminal", "primary_airflow"
+                    )
+
```

Comment:
Again, above code can be simplified by `pint_sum`
```python
hvac_info_dict_b[hvac_id_b]["proposed_supply_cfm"] = pint_sum(
    [terminal_p.get("primary_airflow", ZERO.FLOW)
        for terminal_p in find_all(f'$.buildings[*].building_segments[*].zones[?(@.id == "{zone_id_b}")].terminals[*]',
                    rmi_p)
        for zone_id_b in zones_and_terminal_units_served_by_hvac_sys_dict[hvac_id_b]["zone_list"]
    ],
ZERO.FLOW
)
```

You can replace line 66 with the above code

Reply:
Addressed.

Reply:
As charlie suggested, we can just use sum instead of `pint_sum`

Reply:
Addressed.